components/components.css

```diff
@@ -0,0 +1,856 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+:root {
+  --main-color: #df1b6c;
+  --black: #121212;
```

Comment:
I would suggest you to follow same naming scheme for variables to be consistent. If using --main-color then --black can be --black-color